app/src/reducers/coverPage.js

```diff
@@ -0,0 +1,11 @@
+const initialState = null;
+import { GET_COVER_PAGE_ENTRIES } from '../actions';
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case GET_COVER_PAGE_ENTRIES:
+      return action.payload;
```

Comment:
the only previous state here is either null or the previous list of coverpage elements, which will be replaced anyway. If we add more state here, we need to use object.assign, ofc. but not right now. besides, we are using the same thing on faq, definitions, and mybe some other places too